syntax = "proto3";
package m2.pb;

message Vec2i {
  int32 x = 1;
  int32 y = 2;
}
